Sources/KlaviyoSwift/Klaviyo.swift

```diff
@@ -82,6 +82,11 @@ public struct KlaviyoSDK {
     public func resetProfile() {
         dispatchOnMainThread(action: .resetProfile)
     }
+    
```

Comment:
```suggestion
    /// Resets the badge number on the application icon to 0.
    /// - Parameter defaults: The `UserDefaults` suite where the badge number is persisted.
```
We should try to add documentation whenever possible, especially for client-facing code